calico_containers/tests/st/utils/constants.py

```diff
@@ -17,4 +17,4 @@
 DEFAULT_IPV4_ADDR_2 = "192.168.1.2"
 DEFAULT_IPV4_ADDR_3 = "192.168.1.3"
 DEFAULT_IPV6_POOL_CIDR = "fd80:24e2:f998:72d6::/64"
-LARGE_AS_NUM = "4294967290"
+LARGE_AS_NUM = "4294967200"
```

Comment:
You should make this dotted notation - that'll run through your changes and check it works.